website/blog/2023-03-03-databend-weekly-83.md

```diff
@@ -0,0 +1,182 @@
+  image_url: https://github.com/sundy-li.png
+- name: TCeason
+  url: https://github.com/TCeason
+  image_url: https://github.com/TCeason.png
+- name: wubx
+  url: https://github.com/wubx
+  image_url: https://github.com/wubx.png
+- name: Xuanwo
+  url: https://github.com/Xuanwo
+  image_url: https://github.com/Xuanwo.png
+- name: xudong963
+  url: https://github.com/xudong963
+  image_url: https://github.com/xudong963.png
+- name: youngsofun
+  url: https://github.com/youngsofun
+  image_url: https://github.com/youngsofun.png
+- name: zhang2014
+  url: https://github.com/zhang2014
+  image_url: https://github.com/zhang2014.png
+- name: zhyass
+  url: https://github.com/zhyass
+  image_url: https://github.com/zhyass.png
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+> :loudspeaker: Databend v1.0 is about to be released, and v1.0.0-nightly is already available as a candidate version. https://github.com/datafuselabs/databend/releases/tag/v1.0.0-nightly
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Support WebHDFS Backend
+
+HDFS is a widely used distributed file system for big data, and one of the storage backends supported by Databend. However, in previous implementations, using HDFS depended on Java environment and some specific jars, which might be inconvenient for some users.
+
+To solve this problem, Databend now supports WebHDFS as a storage backend, which facilitates users to interact with existing HDFS storage without Java environment. WebHDFS is a REST API that exposes HDFS operations through HTTP.
+
+```sql
+#> CREATE STAGE IF NOT EXISTS whdfs URL='webhdfs://127.0.0.1:9870/data-files/' CONNECTION=(HTTPS='false');
+Query OK, 0 rows affected (0.020 sec)
+
+#> CREATE TABLE IF NOT EXISTS books (     title VARCHAR,     author VARCHAR,     date VARCHAR );
+Query OK, 0 rows affected (0.030 sec)
+
+#> COPY INTO books FROM @whdfs FILES=('books.csv') file_format=(type=CSV field_delimiter=','  record_delimiter='\n' skip_header=0);
+Query OK, 2 rows affected (0.615 sec)
+
+#> SELECT * FROM books;
++------------------------------+---------------------+------+
+| title                        | author              | date |
++------------------------------+---------------------+------+
+| Transaction Processing       | Jim Gray            | 1992 |
+| Readings in Database Systems | Michael Stonebraker | 2004 |
++------------------------------+---------------------+------+
+2 rows in set (0.044 sec)
+```
+
+If you are interested in learning more about how this is achieved, and how it can improve your analytical experience, please refer to these pull requests:
```

Comment:
Check out these pull requests if you're interested in how it works and improving your analytical experience: